website/templates/project/settings.mako

```diff
@@ -71,28 +71,38 @@
-                                                        value: selectedCategory"></select>
-                        </h5>
-                        <p data-bind="if: !disabled">
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: cancelUpdateCategory"
-                                    class="btn btn-default">Cancel</button>
-                            <button data-bind="css: {disabled: !dirty()},
-                                               click: updateCategory"
-                                    class="btn btn-primary">Change</button>
-                        </p>
-                        <span data-bind="css: messageClass, html: message"></span>
-
-                        <span data-bind="if: disabled" class="help-block">
-                            A top-level project's category cannot be changed.
-                        </span>
-                    </div>
+
+                    <div id="projectSettings" class="panel-body">
+                        <div class="form-group">
+                            <label>Category:</label>
+                            <select data-bind="attr.disabled: disabled,
+                                                         options: categories,
+                                                         optionsValue: 'value',
+                                                         optionsText: 'label',
+                                                         value: selectedCategory"></select>
+                            <span data-bind="if: disabled" class="help-block">
+                              A top-level project's category cannot be changed
+                            </span>
+                        </div>
+                        <div class="form-group">
+                            <label for="title">Title:</label>
+                            <input class="form-control" type="text" maxlength="200" data-bind="value: title,
+                                                                                                      valueUpdate: 'afterkeydown'">
+                            <span class="text-danger" data-bind="validationMessage: title"></span>
+                        </div>
+                        <div class="form-group">
+                            <label for="description">Description:</label>
+                            <textarea data-bind="value: description,
+                                             valueUpdate: 'afterkeydown'",
+                            class="form-control resize-vertical"></textarea>
+                        </div>
+                            <button data-bind="click: updateCategory"
+                            class="btn btn-success">Save Changes</button>
+                            <button data-bind="click: cancelAll"
+                            class="btn btn-default">Cancel</button>
+                            </span>
```

Comment:
In this case the cancel button immediately discards all changes, with asking for confirmation or otherwise notifying the user.

The UI guide is a bit ambiguous wrt how much feedback a cancel/discard-change button should provide. Pinging the ever-talented @caneruguz for clarification on what we want the cancel button to do in this new UI.

![screen shot 2015-09-22 at 11 56 58 am](https://cloud.githubusercontent.com/assets/2957073/10023716/62aaf674-6121-11e5-9910-316b765e29f5.png)

(Button positioning is a known issue and will be changed separately)